src/app/login/login.component.html

```diff
@@ -1,4 +1,4 @@
-<div class="login-pf">
+<div id="login_wrapper" class="login-pf">
 	<span id="badge">
   		<img src="../../assets/img/logo.svg" alt=" logo" />
```

Comment:
There were some existing ids too which are being used in other files. As you created the login component I think for now you would be the better person to review the ids and their usage in this component